pkg/dao/interfaces.go

```diff
@@ -34,3 +34,12 @@ type ExternalResourceDao interface {
 	FetchSignature(url string) (*string, int, error)
 	FetchRepoMd(url string) (*string, int, error)
 }
+
+type MetricsDao interface {
```

Comment:
It is weird! yes, I can see all the metrics!

try the below:
- stop prometheus
- check the scraper configuration is pointing out to the hostname (no localhost)
- check the scrapper interval is a low value (the repo configuration has 3 seconds).
- start prometheus
- do some actions from the frontend (I use to add / delete popular repositories), this will generate information for the metrics.
- await 3 seconds (the scraper interval value).
- refresh prometheus-ui.
- do click on the metrics explorer (it requires to refresh the page); this icon is on the left of `execute` button. I expect here to see all the metrics.

If the above is still failing, let me know, I would need to dig more.

Reply:
Ah, it's working now! I forgot to change `prometheus.yaml` from localhost.

On a similar note, do we need the following? 

1.  add `configs/prometheus.yaml` to `.gitignore`  
2.  add `configs/prometheus.yaml.example`